TASKS/Program.cs

```diff
@@ -9,7 +9,17 @@ namespace TASKS
     internal class Program
     {
         static void Main(string[] args)
-        {
+        {   Console.Write("Введите радиус круга: ");
+            double radius = Convert.ToDouble(Console.ReadLine());
```

Comment:
Класс Covert лучше не использовать без серьезных причин. Лучше так:
```suggestion
            double radius = double.Parse(Console.ReadLine());
```